llama-index-integrations/tools/llama-index-tools-box/pyproject.toml

```diff
@@ -0,0 +1,55 @@
+[build-system]
+build-backend = "poetry.core.masonry.api"
+requires = ["poetry-core"]
+
+[tool.codespell]
+check-filenames = true
+check-hidden = true
+# Feel free to un-skip examples, and experimental, you will just need to
+# work through many typos (--write-changes and --interactive will help)
+skip = "*.csv,*.html,*.json,*.jsonl,*.pdf,*.txt,*.ipynb"
+
+[tool.llamahub]
+contains_example = true
+import_path = "llama_index.tools.box"
+
```

Comment:
Please add `tool.llamahub.class_authors` section:

For example:
```
[tool.llamahub.class_authors]
BoxAIExtractToolSpec = "llama-index"
BoxAIPromptToolSpec = "llama-index"
BoxSearchToolSpec = "llama-index"
BoxSearchByMetadataToolSpec = "llama-index"
BoxTextExtractToolSpec = "llama-index"
```

Feel free to replace `llama-index` above with your Github username.

Reply:
Added